infra/sequelize-orm/modelDbInitializer.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(`${__dirname}/dbConfig.js`)[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  sequelize = new Sequelize(
+    config.database,
+    config.username,
+    config.password,
+    config
+  );
+}
+
+const sourceFolder = path.resolve('src');
+
+fs.readdirSync(path.resolve('src'))
+  .filter((folderName) => folderName !== 'app.js' && folderName !== 'server.js')
```

Comment:
- I am guessing this code loops through all module folders and initializes the db using the various models. 
- - What happens to the /controllers, /config folders etc? I can't quite tell how it differentiates folders with a *model.js and those that don't. Perhaps a comment will help. :)

Reply:
- Yes
- It goes into the folder but doesn't use it another way I think this could be handled is by making this whole file a callable function and then pass in the folder paths to be used or hardcoding the folder paths here that have models e.g
```javascript
['users', 'victims', 'cases', '....otherModules'].map((moFolder) => path.resolve('src', moFolder)).doOtherThings
```

Reply:
If I understand correctly, you are saying a folder without a *model.js file is not impacted. If you have tested this and it works, then no need to change it.. A comment above that loop will suffice. :)

Reply:
updated